daemon/cluster/executor/container/controller.go

```diff
@@ -205,20 +205,14 @@ func (r *controller) Start(ctx context.Context) error {
 	}
 
 	// no health check
-	if ctnr.Config == nil || ctnr.Config.Healthcheck == nil {
+	if ctnr.Config == nil || ctnr.Config.Healthcheck == nil || len(ctnr.Config.Healthcheck.Test) == 0 || ctnr.Config.Healthcheck.Test[0] == "NONE" {
```

Comment:
Will this have the right behavior if `Test` has multiple items, and one of them is not `NONE`?